docs/indexing.rst

```diff
@@ -0,0 +1,123 @@
+Indexing
+========
+
+.. currentmodule:: nvidia.dali
+
+DALI data nodes can be indexed and sliced using familiar Python syntax for array indexing.
+
+Scalar indexing
+~~~~~~~~~~~~~~~
+
+The simplest case is scalar indexing with a constant index::
+
+    raw_files = fn.readers.file(...)
+    third_bytes = raw_files[2]
```

Comment:
Maybe it is better to show the example with the shape in the tensor (and how easy is to extract w and h now) - is it more real life.

Reply:
This sounds like a good idea.